ballerina/soap.bal

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/mime;
+
+# Defines the supported SOAP versions.
+public enum SoapVersion {
```

Comment:
Lets update the doc-comments properly.

Reply:
Fixed in [24188b1](https://github.com/ballerina-platform/module-ballerina-soap/pull/82/commits/24188b1c4975d753810edc6d4f5af71eff23fb03)